### DISCUSSION TITLE
I can't get my student pack

### DISCUSSION THREAD

**Discussion by youssefjoundi** [community/community#113436] [GitHub Education]
### Hi everyone,

I can't get my github student pack
<img width="2243" alt="Screen Shot 2024-03-20 at 1 13 00 PM" src="https://github.com/community/community/assets/59414038/51c4e09d-1988-4716-a396-f43d9282ffce">

**Comment by samus-aran** [comment]
Hi @youssefjoundi 👋 

It looks like your school issues the coupons to its students and faculty via the GitHub Campus program. This means you’ll have to talk to the GitHub administrator for your school and request that they assist you with your Global Campus benefits.

**Comment by digantv** [comment]
Same problem with me
